torch/csrc/autograd/python_function.cpp

```diff
@@ -544,7 +544,7 @@ static void _trace_post_record(
     return;
   }
 
-  node->i_(attr::inplace, is_inplace);
+  node->i_(jit::attr::inplace, is_inplace);
```

Comment:
Build is broken for me. Not sure it's just me. I had to change this.

Reply:
Can you try a rebase to see if that fixes the problem? Is the internal build broken or the OSS one?

Reply:
OSS one. The internal build is also broken. I sent out a diff to fix it.

Reply:
rebasing doesn't solve the problem.